api/types/endpoint.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+type EndpointCreate struct {
+	ExecID string `json:"exec_id"`
+}
+
+type EndpointEvalAttach struct {
+	EvalID string `json:"eval_id"`
+}
+
+type EndpointList struct {
+	Endpoints []Endpoint `json:"endpoints"`
+}
+
+//type Endpoint struct {
+//	ID        string   `json:"id"`
+//	ProjectID string   `json:"project_id"`
+//	Exec      Exec     `json:"exec"`
+//	EvalIDs   []string `json:"eval_ids"`
+//}
+
+type Endpoint struct {
+	ID           string   `json:"id"`
+	ProjectID    string   `json:"project_id"`
+	ExecID       string   `json:"exec_id"`
+	HTTPEndpoint string   `json:"http_endpoint"`
+	EvalIDs      []string `json:"eval_ids"`
+}
+
+type Eval struct {
+	ID   string `json:"id"`
+	Exec Exec   `json:"exec"`
+}
+
+type EvalCreate struct {
+	ExecID string `json:"exec_id"`
+}
```

Comment:
Could we use camelCase here? That's what we're currently using. It might actually make sense to use discuss using snake_case since if we're targeting Python user (since that's the convention there)  but that's a different conversation